### PR TITLE
Bootstrap scenario for gumby 

### PR DESCRIPTION
**Pull request by grimadas**
This pr add a bootstrap scenario for testing tribler bootstrap in Tribler Launch Many module

Nice experiment for bootstrapping a random blob file. You look at the latest run on jenkins:   
https://jenkins-ci.tribler.org/job/validation_experiments/job/validation_bootstrap/15/
image

### CODE REVIEW

**Comment by devos50** [comment]
ok to test

**Comment by qstokkink** [comment]
Is this linked to a Tribler PR?

**Comment by grimadas** [comment]
Link to related tribler pr https://github.com/Tribler/tribler/pull/4453

File: experiments/tunnels/tunnel_module.py [hunk 1]
```diff
@@ -80,11 +81,23 @@ def monitor_downloads(dslist):
...
-                                                                       peer["dtotal"])
+                            pid = unhexlify(peer["id"])
+                            infohash = download.get_def().get_infohash()
+                            if pid not in peer_aggregate:
+                                peer_aggregate[pid] = {}
+                            if infohash not in peer_aggregate[pid]:
+                                peer_aggregate[pid][infohash] = 0
+
+                            peer_aggregate[pid][infohash] += peer["dtotal"]
+                            self._logger.error("Received peers %s (%s, %s) down total: %s, upload: %s, version %s ",
```
Comment (devos50) [inline experiments/tunnels/tunnel_module.py]: Why does this have `error` level? It seems more like `debug` to me.

**Comment by devos50** [comment]
Also, don't forget to rebase 👍

**Review by devos50 — APPROVED** [review]

**Pull request merged**